core/api/serializers.py

```
@@ -2,11 +1,208 @@
+    user must be authenticated to enroll in a contribution
+    
+    """
+    user = UserSerializer(read_only=True)
+    contribution = ContributionSerializer(read_only=True)
+
+    class Meta:
+        model = Enrollment
+        fields = ['id', 'user', 'contribution', 'amount_paid', 'payment_status', 'enrolled_at']
+        read_only_fields = ['id', 'enrolled_at', 'user', 'contribution']
+
+    def create(self, validated_data):
```

Ruff (0.8.2)

Missing return type annotation for public function `create`

(ANN201)